cmake/CMakeLists.txt

```diff
@@ -159,6 +159,9 @@ option(onnxruntime_ENABLE_BITCODE "Enable bitcode for iOS only" OFF)
 # build eager mode
 option(onnxruntime_ENABLE_EAGER_MODE "build ort eager mode")
 
+# build separate library of schemas of (custom) ops used by ORT (for ONNX to MLIR translation)
+option(onnxruntime_BUILD_OPSCHEMA_LIB "Build op schema library" ON)
```

Comment:
I feel it is ok to remove this option and turn it always ON.  Your change has no impact to the core runtime.